lib/KeyboardAwareMixin.js

```diff
@@ -91,6 +97,9 @@ const KeyboardAwareMixin = {
    * @param extraHeight: takes an extra height in consideration.
    */
   scrollToFocusedInput: function (reactNode: Object, extraHeight: number = this.props.extraHeight) {
+    // Android already does this
```

Comment:
Sorry but I come from the iOS world and I'm no Android expert. Are you sure that Android does this for every version since 4.2?

/cc @jblack10101